Description/Descriptor.php

```diff
@@ -51,4 +51,11 @@
     const LINK_REMOVE_REST = 'link.remove.rest';
     const LINK_SHOW_REST = 'link.show.rest';
     const LINK_LIST_REST = 'link.show.rest';
+
+    /**
+     * Permitted children types for this resource.
+     * Value is an array.
+     */
+    const CHILDREN_TYPES = 'children.types';
+    const CHILDREN_ALLOW = 'children.allow';
```

Comment:
See https://github.com/symfony-cmf/resource/issues/25 for a possible alternative to this system of enhancement by using vendor-provided objects as descriptors.